Mod3_Q8.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+
+
+int main()
+{
+    int x;
+    int y;
+    int z;
+    int a;
+    
+    std::cout << "Enter a value for x ";
+    std::cin >> x;
+    std::cout << "Enter a value for y ";
+    std::cin >> y;
+    
+    z = x + y;

```

Comment:
You may want to use variable names that have a bit more meaning. For example here it seems that z represents the sum, so you could name it that.